README.md

```diff
@@ -15,45 +15,49 @@ export AWS_SECRET_ACCESS_KEY="YOUR_SECRET_KEY"
 export AWS_REGION="us-east-1"
 ```
 
+You should write them into the bashrc-file as well, so that they are loaded each time you open a terminal.
+```
+vim ~/.bashrc
+```
 Then you will be able to pull the docker images using the following command:
 
 ```
 docker-compose up
 ```
-Then start the people-api using your IDE or this command
+Then start the people-api using your IDE or the following command. This should also work if `docker-compose up` failed.
 ```
 ./gradlew bootRun
 ```
 
 You will be able to see the people-api service registered in eureka here:
 ```
-http://localhost:8761/
+http://localhost:8081/
```

Comment:
The eureka is something different, you should change that back to 8761.
You'll just be able to see the api registered there if the docker-compose up succeeded and eureka is running.